scripts/qa_crowbarsetup.sh

```diff
@@ -2850,6 +2901,10 @@ function custom_configuration
                 netapp)
                     cinder_netapp_proposal_configuration "0"
                     ;;
+                nfs)
+                    proposal_set_value cinder default "['attributes']['cinder']['volumes'][0]['backend_name']" "'backend_nfs'"
+                    proposal_set_value cinder default "['attributes']['cinder']['volumes'][0]['nfs']['nfs_shares']" "'nfsserver:/srv/nfs/cinder'"
```

Comment:
It seems to work with the defaults. At least it survives a tempest run after: https://github.com/crowbar/crowbar-openstack/pull/869